### ISSUE TITLE
BIBM 2023

### ISSUE THREAD

**Issue by inoue0426** [inoue0426/Paper-reading#2]
# BIBM 2023

- A deep graph convolution network with attention for clustering scRNA-seq data
  - GCN encoder decoder with ZINB loss
  - https://ieeexplore-ieee-org.ezp1.lib.umn.edu/document/10385323
- SEDGCN: Sentiment Enhanced Dual Graph Convolutional Networks for Detecting Adverse Drug Reactions
  - BERT + GCN + HGCN
  - https://ieeexplore-ieee-org.ezp1.lib.umn.edu/document/10385261
- Enhancing Protein Solubility Prediction through Pre-trained Language Models and Graph Convolutional Neural Networks
  - https://github.com/CJ-eeww/PPSol
  - https://ieeexplore.ieee.org/document/10385858
  - Protein embedding with some encoder, esm2 and contactmap graph, then do MLP.
- Predicting Associations between circRNAs and Drug Sensitivity using Heterogeneous Graphs and Graph Attention Networks
  - https://ieeexplore.ieee.org/document/10385513
  - Heterogeneous graph with GAT.
- Multi-omics Sampling-based Graph Transformer for Synthetic Lethality Prediction
  - https://ieeexplore.ieee.org/document/10385953 
  - Omix, GNN, Randomwalk, predict Synthetic Lethality 
- Diffusion-Enhanced Graph Attention Network for Cancer Type Classification
  - https://ieeexplore.ieee.org/document/10386042
  - STRING, TCGA, GAT, diffusion (STRING)
- Imputing single-cell RNA-seq data by graph autoencoder with multi-kernel
  - GCN + AE. 
  - https://ieeexplore.ieee.org/document/10385675
- MoSCHG: Multi-omics Single-cell Classification based on Heterogeneous Graphs and Supervised Contrastive Learning
  - https://ieeexplore.ieee.org/document/10385631
  - Two GCN + two AE. Concat two hidden unites and use it for the input of classifier.
- An Effective Microbial–drug Relation Extraction Model Based on Dual Graph Convolutional Networks
  - https://ieeexplore.ieee.org/document/10385893
  - Dual GCN
- Enhancing Cancer Gene Prediction through Aligned Fusion of Multiple PPI Networks Using Graph Transformer Models
  - https://ieeexplore.ieee.org/document/10385593
  - Several PPI GCN with MLP.
- scGEMOC, A Graph Embedded Contrastive Learning Single-cell Multiomics Clustering Model
  - https://ieeexplore.ieee.org/document/10385267
  - Contrastive learning with ATAC, transcriptome, DDCLoss (?)
- GTDDA: Graph Convolutional Network and Graph Transformer Structure for Drug Repositioning
  - https://ieeexplore.ieee.org/document/10386002
  - Contrastive learning between drug similarity GNN and disease similarity GNN, then compared with known DDAs. then predict drug-disease pair embedding
- Integrating Heterogeneous Biological Networks and Ontologies for Improved Protein Function Prediction with Graph Neural Networks
  - Protein-GO functions prediction from heterogeneous interactions and GO representation usiong GAT and some aggregation
  - https://ieeexplore.ieee.org/document/10385281
- Graph Neural Networks with Geometric Edge Fusion and Point Downsampling for Drug-Target Interaction Prediction
  - https://ieeexplore.ieee.org/document/10385906
  - DTI with 3D Graph and 3D protein coordinates using GNN, MLP, ESM.
  - https://github.com/Hienyriux/GeoPD-DTI
- KSGTN-DDI: Key Substructure-aware Graph Transformer Network for Drug-drug Interaction Prediction
  - DDI, MPNN + positional encoding. 
  - https://ieeexplore.ieee.org/document/10385795
- HGTDG: An Interpretable Heterogeneous Graph Transformer Framework for Cancer Driver Gene Prediction
  - https://ieeexplore.ieee.org/document/10385343
  - Heterogeneous graph transformer for cancer driver gene prediction
  - https://github.com/ZhangLab312/HGTDG
- Integrating Multi-omics Data into A Gated Graph Convolutional Networks for Identifying Cancer Driver Genes and Function Modules
  - https://ieeexplore.ieee.org/document/10385796
  - https://github.com/bionlp-hzau/DGGAT
  - GCN with heterogeneous embedding
